### PR TITLE
Fix serverless offline cache issue

### PR DESCRIPTION
**Pull request by jzc1828**
- [x] Add offline-invalidate plugin to invalidate serverless-offline cache

### CODE REVIEW

**Comment by alhg** [comment]
✅  Verified that the invalidate-cache serverless plugin fixes my issue with serverless not reloading lambda code changes

File: __test__/functions/connection/onConnect.test.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import * as LambdaTester from 'lambda-tester';
 import { handler } from '../../../src/functions/connection/onConnect';
 import * as userFunctions from '../../../src/dynamodb/userDBService';
-import { response } from '../../../src/utils/responseHelper';
 import { createEvent } from '../functionsTestHelpers';
-import { LambdaResponse } from '../../../src/types/response';
+import { FAKE_CONNECTION_ID1 } from '../../testConstants';
 
 const TEST_CONNECTION_ID = 'test-onConnect-connectionId';
```
Comment (alhg) [inline __test__/functions/connection/onConnect.test.ts:7]: Should we also add TEST_CONNECTION_ID to testConstant file?

File: src/functions/connection/onSetUsername.ts [hunk 2]
```diff
@@ -28,18 +26,14 @@ export const handler: Handler = async (event: WebSocketAPIGatewayEvent): Promise
   try {
     if (username) {
       await setUsername(connectionId, username);
-      const wsResponse: WebSocketResponse = createLoginSuccessResponse();
+      const wsResponse = createLoginSuccessResponse();
       await ws.send(wsResponse, connectionId);
-
-      return response(200, `Set username to ${username}`);
+    } else {
+      console.error('onSetUsername: username cannot be empty');
```
Comment (alhg) [inline src/functions/connection/onSetUsername.ts:32]: Should we be sending this error back through websocket instead to let frontend know?

Comment (jzc1828) [reply src/functions/connection/onSetUsername.ts:32]: this is just for the backend logs, any error occurred would be sent to the frontend via line 37 below

File: __test__/functions/connection/onConnect.test.ts (hunk 1 above)
Comment (jzc1828) [reply __test__/functions/connection/onConnect.test.ts:7]: kk

**Pull request merged**